tests/hpc/slurm_master_db.pm

```diff
@@ -63,20 +63,20 @@ sub run {
     }
 
     ## test sbatch
-    my $sbatch = "hpc_slurm_sbatch.sh";
-    script_run("wget --quiet " . data_url($sbatch) . " -O $sbatch");
+    my $sbatch = 'slurm_sbatch.sh';
+    script_run("wget --quiet " . data_url("hpc/$sbatch") . " -O $sbatch");
     assert_script_run("chmod +x $sbatch");
     record_info('meminfo', script_output("cat /proc/meminfo"));
     script_output("sbatch $sbatch");
     ##sbatch SBATCH --time=0-00:01:00
     ## so the worker should wait for the sbatch to finish
     sleep(70);
     upload_logs('/tmp/sbatch1');
-    record_info('sbatch hpc_slurm_sbatch.sh');
+    record_info('INFO', 'sbatch slurm_sbatch.sh');
```

Comment:
it will work like this too , **but** if you will move this line after actual execution you may have something like : 
```suggestion
    record_info('sbatch slurm_sbatch.sh', $output);
```

where `my $output= script_output("sbatch $pbs");`

Reply:
this would simplify test review later on

Reply:
but it is just nice to have so I think I will merge it as it is and you can create another PR with improvement